lib/apiservers/engine/backends/archive.go

```diff
@@ -146,7 +146,14 @@ func (c *Container) importToContainer(op trace.Operation, vc *viccontainer.VicCo
 	mounts := mountsFromContainer(vc)
 	mounts = append(mounts, types.MountPoint{Destination: "/"})
 	writerMap := NewArchiveStreamWriterMap(op, mounts, target)
-	defer writerMap.Close() // This should shutdown all the stream connections to the portlayer.
+	defer func() {
+		// This should shutdown all the stream connections to the portlayer.
+		e1 := writerMap.Close()
+		if err == nil {
```

Comment:
We should be getting error from portlayer via the error returned from io.Copy().  In the proxy, the call to pipeReader.CloseWithErrors() propogates the error back to the backend.  Is this not happening?